examples/resources/kentik-synthetic_test-bulk-create/provider.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    kentik-synthetics = {
+      version = "~> 0.1.0"
```

```suggestion
      version = ">= 0.1.0"
```
`~>` matches only newer patch versions (0.1.x). Please use `>=` to also match minor version (0.x.x) 

Other occurences modified here: https://github.com/kentik/terraform-provider-kentik-synthetics/pull/26/files

---

Addressed with 1c062af49763c49cf3bccd4f7a93ba9973d557c5